pages/docs/learn/durable-execution.mdx

```diff
@@ -0,0 +1,34 @@
+export const description = 'Learn how Durable Execution works with Inngest. '
+
+# How Durable Execution works with Inngest
+
+One of the core features of Inngest is Durable Execution. Durable Execution allows your functions to be fault-tolerant and resilient to failures. The end result is that your code, and therefore, your overall application, is more reliable.
+
+In this guide, we'll cover the basics of Durable Execution, how it works, and how it works with Inngest functions.
+
+{/* Note - this page is written a specific way for search optimization */}
+## What is Durable Execution?
+
+Durable Execution is a fault-tolerant approach to executing code that handles failures and interruptions gracefully through automatic retries and state persistence. This means that your code can continue to run even if there are issues like network failures, timeouts, infrastructure outages, and other transient errors.
+
+Durable functions can throw errors or exceptions and automatically retry, resuming execution from the point of failure. Durable functions are designed to be long-running and stateful, meaning that they can persist state across function invocations and retries.
```

Comment:
"Durable functions" are a new term. Should we add something like:

```suggestion
At the foundation of Durable Execution are the so-called "durable functions". Durable functions can throw errors or exceptions and automatically retry, resuming execution from the point of failure. They are designed to be long-running and stateful, meaning that they can persist state across function invocations and retries.
```

Reply:
Additionally, it feels like this paragraph belongs at the end of this section (after "Some key aspects" which expands on the previous paragraph).